test/lib/code_corps/policy/user_skill_test.exs

```diff
@@ -11,21 +11,19 @@ defmodule CodeCorps.Policy.UserSkillTest do
       user = build(:user, admin: true)
       changeset = %UserSkill{} |> create_changeset(%{})
 
-      assert create?(user, changeset) 
+      assert create?(user, %{"user_id" => user.id}) 
```

Comment:
You're not using the changeset here anymore. You can likely also remove `import CodeCorps.UserSkill, only: [create_changeset: 2]` from the top of the file.

Sorry I didn't catch this on my previous pass.

Reply:
oops.  fixed.